### DISCUSSION TITLE
开源治理：企业采用开源的激进与保守 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#45] [Announcements]
# 开源治理：企业采用开源的激进与保守 - 「开源之道」

https://opensourceway.community/posts/opensource_enterprise_guide/the-art-of-balance-for-adopt-open-source/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

对于企业而言, 反而更加简单,
有利可图, 那就弄,
无利可图, 就无视...

所有激进还是保守, 都不过是事后的状态总结,
决策那一瞬间, 都是被逼的....

看看 M$ 这几十年对 Linux 们的态度就知道了...

**Reply by lijiangsheng1** [reply]
未雨绸缪，有危机感与就这样吧，守住现有的就好，二者是矛盾的，就和人体内那一对to be or not to be的斗争一样。
